src/ec/packages/ec-component-table/__snapshots__/table.test.js.snap

```diff
@@ -1102,6 +1102,216 @@ exports[`EC - Table Default renders correctly with row extra classes 1`] = `
+        data-test-another=""
+      >
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Job title"
+        >
+          Administators in Financial rules appliable to the EU budget
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="EFSI finance approved by EIB"
+        >
+          AD7
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Location"
+        >
+          Brussels (Belgium), Luxembourg (Luxembourg), Strasbourg (France)
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Organization"
+        >
+          European Commission
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Type of contract"
+        >
+          Permanent official
+        </td>
+      </tr>
+      <tr
+        class="ecl-table__row row-extra-class row-extra-class row-extra-class row-extra-class"
+        data-test=""
+        data-test-another=""
+      >
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Job title"
+        >
+          Corporate Support Officer
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="EFSI finance approved by EIB"
+        >
+          FG IV
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Location"
+        >
+          Prague (Czech Republic)
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Organization"
+        >
+          European Commission
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Type of contract"
+        >
+          Permanent official
+        </td>
+      </tr>
+      <tr
+        class="ecl-table__row row-extra-class row-extra-class row-extra-class row-extra-class row-extra-class"
+        data-test=""
+        data-test-another=""
+      >
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Job title"
+        >
+          Policy Officer - Clean Energy For All Europeans
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="EFSI finance approved by EIB"
+        >
+          FG II, FG III, FG IV
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Location"
+        >
+          Vigo (Spain)
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Organization"
+        >
+          EU-LISA
+        </td>
+        <td
+          class="ecl-table__cell"
+          data-ecl-table-header="Type of contract"
+        >
+          Seconded National Expert (SNE)
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <!-- ################################################################################################################
```

Comment:
Looking at the snapshot (not only this one, but this is specifically too big), only this part is actually useful for the assertion https://github.com/ec-europa/ecl-twig/pull/441/files#diff-516d297d2ac695f7864b6efaf0e8e1aeR1298-R1311

If you find a way to either assert only for message string value or this particular dom element it will improve the focus and usability of this assertion.